roiextractors/extractors/suite2p/suite2psegmentationextractor.py

```diff
@@ -53,6 +51,17 @@ def get_accepted_list(self):
     def get_rejected_list(self):
         return np.where(self.iscell[:,0]==0)[0]
 
+    def _summary_image_read(self, bstr='meanImg'):
+        img = None
+        if bstr in self.ops:
+            if bstr == 'Vcorr' or bstr == 'max_proj':
+                img = np.zeros((self.ops['Ly'], self.ops['Lx']), np.float32)
+                img[self.ops['yrange'][0]:self.ops['yrange'][-1],
+                self.ops['xrange'][0]:self.ops['xrange'][-1]] = self.ops[bstr]
+            else:
+                img = self.ops[bstr]
+        return img
```

Comment:
I don't really understand what's going on here

Reply:
are you trying to change the data type? Or flip the image?

Reply:
this is how its implemented in the suite2p nwbwrite method: they have selected a subframe within the self.ops['Vcorr'] image to assign to the correlation image. This subset is defined by the keys 'xrange' and 'yrange' in self.ops. 
This is their implementation: https://github.com/MouseLand/suite2p/blob/main/suite2p/io/nwb.py#L272

Reply:
oh, I see. OK, I think they were storing images from multiple planes side-by-side and then pulling the pixels corresponding to the plane of interest